File: C/Estrutura Repetitiva/Para/Media_ponderada.c
#include <stdio.h>
#include <string.h>


int main ()
{
    int N, i;
    double A, B, C, media;

    printf ("Quantos casos voce vai digitar? ");
    scanf ("%d", &N);


    for (i = 0; i < N; i++)
    {
        printf("Digite tres numeros:\n");
        scanf ("%lf", &A);
        scanf ("%lf", &B);
        scanf ("%lf", &C);
        media = ((A * 2)+(B * 3)+(C * 5)) / 10;
        printf ("MEDIA = %.1lf\n", media);
            }


    return 0;
}

File: Python/Estrutura Repetitiva/Para/Media_ponderada.py
N: int; i: int
a: float; b: float; c: float; media: float

N = int(input("Quantos casos voce vai digitar? "))

for i in range(1, N + 1):
    print(f"Digite tres numeros: ")
    a = float(input())
    b = float(input())
    c = float(input())
    media = ((a * 2) + (b * 3) + (c * 5)) / 10
    print(f"MEDIA = {media:.1f}")